Rubberduck.Core/UI/UnitTesting/TestExplorerViewModel.cs

```diff
@@ -215,6 +231,15 @@ private void HandleTestCompletion(object sender, TestCompletedEventArgs e)
             Tests.Refresh();
         }
 
+        public IRewritingManager RewritingManager { get; }
+
+        public string UnignoreTestLabel => RubberduckUI.TestExplorer_TestToggle_Unignore;
+        public string IgnoreTestLabel => RubberduckUI.TestExplorer_TestToggle_Ignore;
+
+        private TestMethod _mousedOverTestMethod => ((TestMethodViewModel)SelectedItem).Method;
```

Comment:
if we check for `SelectedItem` being null, this shouldn't assume it is not. The reason for that is that we can't ensure this is only evaluated in `Display(Uni|I)gnoreTestLabel`. It might be easier to have something like `SelectedItemIgnoreMatches(bool ignoreState) => (((TestMethodViewModel)SelectedItem)?.Method.IsIgnored ?? !ignoreState) == ignoreState;`. This is a bit... dense, though, so a proper method would probably be best ;)

Reply:
I'll need more breadcrumbs to follow and understand this.